vars/landing.tfvars

```diff
@@ -0,0 +1,6 @@
+landing_account_id = "335823981503"
+ap_accounts        = {
+  data = "593291632749"
+  dev  = "525294151996"
+  prod = "312423030077"
+}
```

Comment:
I'd rename this file to something like __ap_accounts.tfvars__ (your choice).  You could also just add the landing value to the `ap_accounts` map as it has no significance here (I think 🤷‍♂)???

Reply:
See my reply to your comment (first one) :)

Reply:
I'm going to rename it